unit_tests/src/test_host_list_context_string.cpp

```diff
@@ -0,0 +1,83 @@
+#include <catch2/catch.hpp>
+
+#include <string_view>
+#include <sstream>
+#include <vector>
+
+#include "irods/irods_plugin_base.hpp"
+#include "irods/filesystem/path.hpp"
+#include "irods/private/irods_is_in_host_list.hpp"
+
+TEST_CASE("host list tests", "[detached_mode_host_list]")
+{
+    const std::string_view resource_hostname = "myhost";
+
+    SECTION("not in host list")
+    {
+        irods::plugin_property_map prop_map;
```

Comment:
Optional: Instantiating the property map could happen at the top of the `TEST_CASE` since each `SECTION` is instantiating and using it.

Reply:
done